app/src/main/java/org/xwiki/android/sync/contactdb/ContactOperations.kt

```diff
@@ -0,0 +1,183 @@
+package org.xwiki.android.sync.contactdb
+
+import android.accounts.Account
+import android.content.ContentProviderOperation
+import android.content.ContentResolver
+import android.content.ContentUris
+import android.provider.ContactsContract
+import org.xwiki.android.sync.Constants
+import org.xwiki.android.sync.bean.XWikiUserFull
+import android.content.ContentValues
+
+
+/**
+ * When we first add a sync adapter to the system, the contacts from that
+ * sync adapter will be hidden unless they're merged/grouped with an existing
+ * contact.  But typically we want to actually show those contacts, so we
+ * need to mess with the Settings table to get them to show up.
+ *
+ * @param resolver Will need to insert new data into system db
+ * @param account the Account who's visibility we're changing
+ * @param visible true if we want the contacts visible, false for hidden
+ *
+ * @since 0.4
```

Comment:
Is this still accurate ?

Reply:
This method was copypasted from old class and adapted to kotlin

Reply:
I understood that but then I see in your code

    import static org.xwiki.android.sync.contactdb.ContactOperationsKt.setAccountContactsVisibility;

which seems to indicate that the class name changed from ContactOperations to ContactOperationsKt.

Reply:
Because in fact I replace it from Java into kotlin class

Reply:
Which means this is a new class the the since is wrong.

Reply:
ok, will fix

Reply:
Fixed